odoo/orm/fields_properties.py

```diff
@@ -350,11 +355,12 @@ def _add_default_values(self, env, values):
 
         return properties_list_values
 
-    def _get_properties_definition(self, record):
+    def _get_properties_definition(self, record: BaseModel) -> PropertiesDefinition | None:
```

Comment:
Instances of `PropertiesDefinitions` are _fields_, not field values.  The values of such a field is a list of dicts.